src/main/java/org/broadinstitute/consent/http/resources/DataRequestElectionResource.java

```diff
@@ -78,6 +78,13 @@ public Response createDataRequestElection(@Context UriInfo info, Election rec,
             emailNotifierService.sendNewCaseMessageToList(darVotes, accessElection);
             uri = info.getRequestUriBuilder().build();
         } catch (Exception e) {
+            try {
+                if (Objects.nonNull(accessElection)) {
+                    api.deleteElection(requestId, accessElection.getElectionId());
+                }
+            } catch (Exception e2) {
+                logger().warn("Error deleting created access election: ", e2);
+            }
```

Comment:
This is essentially cleanup code so we don't leave behind elections with no votes or an Access election without an associated RP election